examples/mundialis_example.py

```diff
@@ -31,8 +39,9 @@
 datacube = con.imagecollection("nc_spm_08.modis_lst.strds.MOD11B3.A2016306.h11v05.single_LST_Day_6km")
```

Comment:
do not calculate NDVI from LST
use as strds utm32n.openeo_bolzano.strds.openeo_bolzano_S2